Makefile

```diff
@@ -145,12 +140,31 @@ endif
 clean-scorecard:
 	- $(call scorecard-cleanup); cleanup
 
+ifneq ($(and $(SCORECARD_REGISTRY_SERVER),$(SCORECARD_REGISTRY_USERNAME),$(SCORECARD_REGISTRY_PASSWORD)),)
+SCORECARD_ARGS := --pull-secret-name registry-key --service-account cryostat-scorecard
+endif
+
+define scorecard-setup
+@$(CLUSTER_CLIENT) get namespace $(SCORECARD_NAMESPACE) >/dev/null 2>&1 &&\
+	echo "$(SCORECARD_NAMESPACE) namespace already exists, please remove it with \"make clean-scorecard\"" >&2 && exit 1 || true
+$(CLUSTER_CLIENT) create namespace $(SCORECARD_NAMESPACE)
+cd internal/images/custom-scorecard-tests/rbac/ && $(KUSTOMIZE) edit set namespace $(SCORECARD_NAMESPACE)
+$(KUSTOMIZE) build internal/images/custom-scorecard-tests/rbac/ | $(CLUSTER_CLIENT) apply -f -
+if [ -n "$(SCORECARD_ARGS)" ]; then \
```

Comment:
I'm getting a Make error here. Are we able to use `ifneq` here instead of Bash?

Reply:
oh yeh I just updated that to silent the entire if. I tried `ifneq` but got syntax error all over...

Reply:
Worked here: https://github.com/tthvo/cryostat-operator/actions/runs/5074021438/jobs/9113746052

Just need that k8s support for scorecard :))